NEWS.md

```diff
@@ -237,6 +237,36 @@ Outstanding ones only.
         p C.ancestors #=> [C, M1, M2, Object, Kernel, BasicObject]
         ```
 
+    * Module#public, Module#protected and Module#private methods now accept single
+      array (or array like) argument with a list of method names.
+      [[Feature #17314]]
+
+        ```ruby
+        class Foo
+          def foo; end
+          def bar; end
+
+          private [:foo, :bar]
+        end
+        ```
+
```

Comment:
Please remove. We keep `NEWS` short, people refer to the linked issue / doc if they want more details.